project/plugins.sbt

```diff
@@ -19,6 +19,7 @@ addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.3.5")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.1.5")
 addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.5.1")
 addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.12")
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
```

Comment:
Do they have hard coded mapping for Dataflow, BigQuery JavaDocs etc.?
TBH those are nice to have I don't mind not having the link & keep this simple.

Reply:
The plugin resolves the javadoc.io links for them ... so when we have `PCollection` in a scaladoc ... users can actually jump to the reespective javadoc

Reply:
I'm just curious how does it know the URL prefix for say Beam JavaDoc?

Reply:
https://github.com/ThoughtWorksInc/sbt-api-mappings/tree/3.0.x/src/main/scala/com/thoughtworks/sbtApiMappings
So looks like a bunch of hard coded ones including JDK lib, spark, play, but that's about it. 🤷‍♂

Reply:
for most of the deps will use the sonatype mapping rule.